cmd/ontap-cvo.go

```diff
@@ -487,22 +484,8 @@ func performHttpPost(username string, password string, url string, requestBody [
 	return resp.StatusCode, responseBody
 }
 
-// TODO: Retrieve the svmInfo this must query the master configmap that exists in the DAS namespace.
-// This configmap COULD change, so could put this elsewhere and have it be one call.
-// How the mapping here works with naming will need to be settled on right now this will fail
-func getSvmInfo(client *kubernetes.Clientset, whichSVM string) svmInfo {
-	svmName := ""
-	svmUUID := ""
-	svmURL := ""
-	svmInformation := svmInfo{
-		svmName: svmName,
-		svmUUID: svmUUID,
-		svmUrl:  svmURL,
-	}
-	return svmInformation
-}
-
 // Format JSON data helper function
+// TODO: is this needed or should this be removed?
```

Comment:
I am now thinking that it can be removed, given that we have like all our functionality and didnt need it

Reply:
Ill remove it now